Unable to initialise device_tracker.bluetooth_le_tracker, error installing dependency gattlib

Make sure you are running the latest version of Home Assistant before reporting an issue.

You should only file an issue if you found a bug. Feature and enhancement requests should go in [the Feature Requests section](https://community.home-assistant.io/c/feature-requests) of our community forum:

**Home Assistant release (`hass --version`):**
0.45.1

**Python release (`python3 --version`):**
Python 3.6.1

**Component/platform:**
Device tracker / Bluetooth LE Tracker

**Description of problem:**
Home Assistant running on docker on unRaid 6.3.5 can't install required gattlib

**Problem-relevant `configuration.yaml` entries and steps to reproduce:**
```yaml
- platform: bluetooth_le_tracker
- platform: bluetooth_tracker
```

**Traceback (if applicable):**
```bash
2017-06-01 13:37:21 ERROR (<concurrent.futures.thread.ThreadPoolExecutor object at 0x2ac09940fd68>_0) [homeassistant.util.package] Unable to install package gattlib==0.20150805: Command "/usr/local/bin/python -u -c "import setuptools, tokenize;__file__='/tmp/pip-build-pr8vs_y4/gattlib/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmp/pip-59cbsf21-record/install-record.txt --single-version-externally-managed --compile --home=/tmp/tmprm5hszgq" failed with error code 1 in /tmp/pip-build-pr8vs_y4/gattlib/
2017-06-01 13:37:21 ERROR (MainThread) [homeassistant.setup] Not initializing device_tracker.bluetooth_le_tracker because could not install dependency gattlib==0.20150805
2017-06-01 13:37:21 ERROR (MainThread) [homeassistant.setup] Unable to prepare setup for platform device_tracker.bluetooth_le_tracker: Could not install all requirements.
2017-06-01 13:37:21 ERROR (MainThread) [homeassistant.components.device_tracker] Error setting up platform bluetooth_tracker
Traceback (most recent call last):
  File "/usr/src/app/homeassistant/components/device_tracker/__init__.py", line 161, in async_setup_platform
    disc_info)
  File "/usr/local/lib/python3.6/concurrent/futures/thread.py", line 55, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/usr/src/app/homeassistant/components/device_tracker/bluetooth_tracker.py", line 64, in setup_scanner
    for dev in discover_devices():
  File "/usr/src/app/homeassistant/components/device_tracker/bluetooth_tracker.py", line 42, in discover_devices
    lookup_class=False)
  File "/config/deps/bluetooth/bluez.py", line 26, in discover_devices
    device_id = _bt.hci_get_route()
OSError: [Errno 97] Address family not supported by protocol
```

**Additional info:**
I'm using official's @balloob unRaid docker image. If trying to install manually gattlib from inside the docker by using pip3 install gattlib I get the following
```bash
root@unBeast:/usr/src/app# pip3 install gattlib   
Collecting gattlib
  Using cached gattlib-0.20150805.tar.gz
Building wheels for collected packages: gattlib
  Running setup.py bdist_wheel for gattlib ... error
  Complete output from command /usr/local/bin/python -u -c "import setuptools, tokenize;__file__='/tmp/pip-build-ui3frnkq/gattlib/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" bdist_wheel -d /tmp/tmp96130q8lpip-wheel- --python-tag cp36:
  running bdist_wheel
  running build
  running build_ext
  building 'gattlib' extension
  creating build
  creating build/temp.linux-x86_64-3.6
  creating build/temp.linux-x86_64-3.6/src
  creating build/temp.linux-x86_64-3.6/src/bluez
  creating build/temp.linux-x86_64-3.6/src/bluez/lib
  creating build/temp.linux-x86_64-3.6/src/bluez/attrib
  creating build/temp.linux-x86_64-3.6/src/bluez/src
  creating build/temp.linux-x86_64-3.6/src/bluez/src/shared
  creating build/temp.linux-x86_64-3.6/src/bluez/btio
  gcc -pthread -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -fPIC -DVERSION="5.25" -I/usr/include/glib-2.0 -I/usr/lib/x86_64-linux-gnu/glib-2.0/include -Isrc/bluez -I/usr/local/include/python3.6m -c src/gattservices.cpp -o build/temp.linux-x86_64-3.6/src/gattservices.o
  cc1plus: warning: command line option ‘-Wstrict-prototypes’ is valid for C/ObjC but not for C++
  In file included from src/gattservices.cpp:12:0:
  src/gattlib.h:11:33: fatal error: boost/python/list.hpp: No such file or directory
   #include <boost/python/list.hpp>
                                   ^
  compilation terminated.
  error: command 'gcc' failed with exit status 1
  
  ----------------------------------------
  Failed building wheel for gattlib
  Running setup.py clean for gattlib
Failed to build gattlib
Installing collected packages: gattlib
  Running setup.py install for gattlib ... error
    Complete output from command /usr/local/bin/python -u -c "import setuptools, tokenize;__file__='/tmp/pip-build-ui3frnkq/gattlib/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmp/pip-0610butn-record/install-record.txt --single-version-externally-managed --compile:
    running install
    running build
    running build_ext
    building 'gattlib' extension
    creating build
    creating build/temp.linux-x86_64-3.6
    creating build/temp.linux-x86_64-3.6/src
    creating build/temp.linux-x86_64-3.6/src/bluez
    creating build/temp.linux-x86_64-3.6/src/bluez/lib
    creating build/temp.linux-x86_64-3.6/src/bluez/attrib
    creating build/temp.linux-x86_64-3.6/src/bluez/src
    creating build/temp.linux-x86_64-3.6/src/bluez/src/shared
    creating build/temp.linux-x86_64-3.6/src/bluez/btio
    gcc -pthread -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -fPIC -DVERSION="5.25" -I/usr/include/glib-2.0 -I/usr/lib/x86_64-linux-gnu/glib-2.0/include -Isrc/bluez -I/usr/local/include/python3.6m -c src/gattservices.cpp -o build/temp.linux-x86_64-3.6/src/gattservices.o
    cc1plus: warning: command line option ‘-Wstrict-prototypes’ is valid for C/ObjC but not for C++
    In file included from src/gattservices.cpp:12:0:
    src/gattlib.h:11:33: fatal error: boost/python/list.hpp: No such file or directory
     #include <boost/python/list.hpp>
                                     ^
    compilation terminated.
    error: command 'gcc' failed with exit status 1
    
    ----------------------------------------
Command "/usr/local/bin/python -u -c "import setuptools, tokenize;__file__='/tmp/pip-build-ui3frnkq/gattlib/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmp/pip-0610butn-record/install-record.txt --single-version-externally-managed --compile" failed with error code 1 in /tmp/pip-build-ui3frnkq/gattlib/
```

--- a/virtualization/Docker/setup_docker_prereqs
+++ b/virtualization/Docker/setup_docker_prereqs
@@ -20,7 +20,7 @@ PACKAGES=(
   # homeassistant.components.device_tracker.nmap_tracker
   nmap net-tools libcurl3-dev
   # homeassistant.components.device_tracker.bluetooth_tracker
-  bluetooth libglib2.0-dev libbluetooth-dev
+  bluetooth libglib2.0-dev libbluetooth-dev libboost-python1.62-dev
   # homeassistant.components.device_tracker.owntracks
   libsodium13
   # homeassistant.components.zwave

